infra/test_repos.py

```diff
@@ -48,6 +50,7 @@
     ExampleRepo(project_name='curl',
                 oss_repo_name='curl',
                 git_repo_name='curl',
+                image_location='/src',
```

Comment:
Sorry, I don't quite understand why this change was needed? Can you please explain?

Reply:
This is required for testing. It is needed at line 93 of build_specified_commit_test.py